Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAuditdPlugin.py

```diff
@@ -92,12 +95,23 @@ def GetAuomOutputConfPath(WorkspaceId):
 def GetTmpAuditRulesPath(WorkspaceId):
     return os.path.join(GetTmpDir(WorkspaceId), "auditd_plugin.rules")
 
+def GetTmpAuditLoadedRulesPath(WorkspaceId):
+    return os.path.join(GetTmpDir(WorkspaceId), "auditd_loaded.rules")
+
 def Set(WorkspaceId, Ensure):
     if not IsValidWorkspaceId(WorkspaceId):
         return [1]
 
-    (audit_version, plugin_state,
-     actual_audit_rules, actual_outconf, actual_plugin_conf) = GetState(WorkspaceId)
+    if IsSudoScriptOutOfDate():
+        LG().Log(LOG_INFO, "Updating sudo script")
+        try:
+            WriteFile(SUDO_SCRIPT, ReadFile(RESOURCE_SUDO_SCRIPT))
```

Comment:
This line "WriteFile(SUDO_SCRIPT, ReadFile(RESOURCE_SUDO_SCRIPT))" does the copy. I already use ReadFile and WriteFile elsewhere and didn't want to import shutil just to do the copy.